src/main/java/internet/market/controllers/InjectUserDataController.java

```diff
@@ -0,0 +1,26 @@
+package internet.market.controllers;
+
+import internet.market.lib.Injector;
+import internet.market.model.User;
+import internet.market.service.UserService;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class InjectUserDataController extends HttpServlet {
```

Comment:
The checklist said "Use single separate controller to inject data", so maybe you'd need to combine InjectUserDataController & InjectProductDataController

Reply:
okay thanks